server/src/user/test/user.e2e-spec.ts

```diff
@@ -87,4 +87,42 @@ describe('UserController (e2e)', () => {
       });
     });
   });
+
+  describe('DELETE /me/?houseId=:houseId', () => {
+    // const userId = 106;
```

Comment:
If you don't use it, you can delete the line!

Reply:
Fix it. 
c7fbf3878fea460517f6ab3c4c2edbbeceb299ca